sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ece-request/ece-process-integration.scheduler.ts

```diff
@@ -0,0 +1,49 @@
+import { InjectQueue, Process, Processor } from "@nestjs/bull";
+import { ECEFileService } from "@sims/integrations/institution-integration/ece-integration";
+import { QueueService } from "@sims/services/queue";
+import { QueueNames } from "@sims/utilities";
+import { InjectLogger, LoggerService } from "@sims/utilities/logger";
+import { Job, Queue } from "bull";
+import { BaseScheduler } from "../../base-scheduler";
+import {
+  GeneratedDateQueueInDTO,
+  ECEProcessResultQueueOutDTO,
```

Comment:
we are not using `QueueInDTO or QueueOyrDTO` for queues

Reply:
We are still using `QueueInDTO` when needed. The object received by the queue can be considered a DTO, as per our last conversation.